cmd/metrics-node-sampler/integration/testdata/expected.json

```diff
@@ -53,8 +51,8 @@
 			"nodeName": "",
 			"avgCPUCoresNanoSec": "390",
 			"avgMemoryBytes": "310",
-			"totalOOMCount": "211",
-			"totalOOMKillCount": "29",
+			"oomCount": "28",
```

Comment:
This change in the test seems weird. How do we go from 211 totalOOMCount to 28?

Reply:
Before it was a total, which didn't work for tests because the value was dependent on when the list request is made.  Now it is the total during the "5m" window (or 5 sample window for the tests).  Because there are only 5 samples which are rotated you get a stable value.